app/routes/password/reset.js

```diff
@@ -24,10 +24,10 @@ export default Route.extend({
         data: {
           token: get(this, 'controller.token'),
           password,
-          passwordConfirmation
+          "password-confirmation": passwordConfirmation
```

Comment:
Curious about this change.  Is this a stylistic thing?  Just asking for future ref.